conftest.py

```diff
@@ -74,9 +77,7 @@ def run(self, **kwargs):
         -------
         docker.Container
         """
-        all_kwargs = {}
-        all_kwargs.update(self.kwargs)
-        all_kwargs.update(kwargs)
+        all_kwargs = self.kwargs | kwargs
```

Comment:
Oh I had never seen this before, today I learned something new!

Reply:
Yep, it's Python 3.9 feature.
https://docs.python.org/3/whatsnew/3.9.html#dictionary-merge-update-operators

I've never heard any complaints, that we use latest python version, so I think it's reasonable to continue to use new features sometimes :)